dt-posts/posts.php

```diff
@@ -1301,12 +1307,21 @@ public static function search_viewable_post( string $post_type, array $query, bo
             return $fields_sql;
         }
 
+        // If detected, ensure to also query by map bound lat/lng coordinates.
+        $map_bounds_join_sql = '';
+        $map_bounds_where_sql = '';
+        $has_map_bounds = isset( $map_bounds['ne'], $map_bounds['ne']['lat'], $map_bounds['ne']['lng'], $map_bounds['sw'], $map_bounds['sw']['lat'], $map_bounds['sw']['lng'] );
+        if ( $has_map_bounds ) {
+            $map_bounds_join_sql = 'LEFT JOIN wp_dt_location_grid_meta as location_grid_meta ON ( location_grid_meta.post_id = p.ID )';
```

Comment:
@kodinkat watch out for the hard coded table names. should be $wpdb->dt_location_grid_meta